hokusai/lib/command.py

```diff
@@ -19,22 +19,34 @@ def wrapper(*args, **kwargs):
           sys.exit(0)
         else:
           sys.exit(result)
-      except HokusaiError as e:
-        print_red(e.message)
-        sys.exit(e.return_code)
       except SystemExit:
         raise
       except KeyboardInterrupt:
         raise
-      except (CalledProcessError, Exception) as e:
+      except HokusaiError as e:
+        print_red("ERROR: %s" % e.message)
+        sys.exit(e.return_code)
+      except CalledProcessError as e:
         if get_verbosity() or os.environ.get('DEBUG'):
           print_red(traceback.format_exc())
         else:
           print_red("ERROR: %s" % str(e))
         if hasattr(e, 'output') and e.output is not None:
-          print(e.output.decode('utf-8'))
-        elif hasattr(e, 'message') and e.message is not None:
-          print(e.message.decode('utf-8'))
+          if str == bytes:
```

Comment:
I am sorry there has to be conditionals based on python versions! 😭 Nice catch on handling both.

Not blocking but maybe someday we can encapsulate some of the python checks out into their own function since it is needed in a few places.

Reply:
Yeah the string / bytes unicode changes between Python 2 / 3 are doing my head in 😓   This seems to be the most concise way to check it.  Good call on a single function... i.e. `which_python()` that returns `2` or `3`.  That would make my life a lot easier!